sass/components/html/page-header-complete.html

```diff
@@ -0,0 +1,54 @@
+<div class="page-header">
+  <nav class="page-navigation" role="navigation">
+    <div class="container-fluid">
+      <nav class="breadcrumb breadcrumb-processed breadcrumb--collapsible" role="navigation" aria-label="breadcrumbs">
+        <ol class="breadcrumb__segments-wrapper">
+          <li class="breadcrumb__segment breadcrumb__segment--first">
+            <a href="#" class="breadcrumb__link is-internal" rel="home">
+              <span class="breadcrumb__text">European Commission</span>
+            </a>
+          </li>
+          <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Digital single market</span>
+            </a>
+          </li>
+          <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Strategy</span>
+            </a>
+          </li>
+           <li class="breadcrumb__segment breadcrumb__segment--last">
+            <a href="#" class="breadcrumb__link">
+              <span class="breadcrumb__text">Newsroom</span>
+            </a>
+          </li>
+        </ol>
+      </nav>
+    </div>
+  </nav>
+  <div class="container-fluid page-header__site-identification">
+    <h3>Digital single market</h3>
+  </div>
+  <div class="container-fluid page-header__hero-title">
+    <div class="row padding-reset">
+      <div class="col-lg-9">
+        <div class="meta meta--header">
+          <span class="meta__item meta__item--type">news article</span>
+          <span class="meta__item">
+            <span class="date-display-single" property="dc:date" datatype="xsd:dateTime" content="2015-07-06T17:44:28+02:00">6 July 2015</span>
+          </span>
+          <span class="meta__item">Brussels</span>
+        </div>
+        <div class="field field--title">
```

Comment:
you are referring to this, i think: field--title
this is what our customization of the theme_field is doing, we always have that class created as a modifier  of the field component with the field name, i think we have this many times in the styleguide.
I do agree that speaking about the styleguide, this doesn't have to reflect what we have out of the europa theme implementation for drupal, so yes, i will remove this class but again, i invite to create a ticket about cleaning up those classes in the styleguide templates because i believe this is not the only occurrence